ci/pipeline/determine_tests_to_run.py

```diff
@@ -366,9 +366,11 @@ def get_commit_range():
     print(output_string, file=sys.stderr)
 
     # Used by buildkite log format
+    pairs = [item.split("=") for item in output_string.split(" ")]
+    affected_vars = [key for key, affected in pairs if affected == "1"]
     if args.output.lower() == "json":
-        pairs = [item.split("=") for item in output_string.split(" ")]
-        affected_vars = [key for key, affected in pairs if affected == "1"]
         print(json.dumps(affected_vars))
+    if args.output.lower() == "envvars":
+        print(" ".join(affected_vars))
```

Comment:
this is not printing the same `affected_vars` but without `=`. It still needs to print `output_string` when it is `envvars`.

Reply:
I don't get what you mean ;), 

If the output_string is A=1 B=0

This is printing A (throwing away B), which is what we want right?

Reply:
I mean you are changing how it works.

when `output == "envvars"` (which is the default, it was originally printing `A=1 B=0` (`output_string`), now it is printing `A B` (`" ".join(affected_vars)`). I think this will break civ1 's usage.

Reply:
ahh good catch, i didn't notice envvars is its default value

Reply:
If I were you, I would add a new supported `output` flag value that is called maybe `tags`.